### ISSUE TITLE
Would it be possible to have isoelasticity lines for volume-deformation plots?

### ISSUE THREAD

**Issue by MartaUrb** [ZELLMECHANIK-DRESDEN/ShapeOut#235]
Do we have volume information for the E modulus from numerical simulations? It would be cool to be able to plot the volume against deformation with isoelasticity lines.

Although I  am starting to doubt that the rotation of the contour gives an accurate estimate of volume. From the numerical simulations in the square channel it seems like the cell is actually taking some shape that has more of a square with rounded edges cross section rather than circular one in the axis perpendicular to the flow direction....

**Comment by paulmueller** [comment]
Yes, it should be possible to compute deformation-vs-volume isoelasticity lines.

Indeed, the shape of the cross-section perpendicular to the channel axis might not be rotationally symmetric. However, I think that the effect on volume will be small. Which numerical simulations are you referring to?

The only thing that I am unsure about are the conversion rules of deformation-volume isoelasticity lines for different flow rates, channel widths, or medium viscosities.

@chrherold How big would the effort be to extract the volume in addition to the area from the simulations? I guess they would have to be run again...

**Comment by chrherold** [comment]
I do not have the volume information but the guys who did the simulation should. 

One would have to decide to either 
- take the volume as constant during simulation (which is assumed for the simulations)
This way one only needs to match the deformation points along the deformation path with the original volume which I guess could be done very fast.

- consider the "real life" effect that the convex hull has on the data as is done for the area. For this, one needs the shapes of in every data point which as far as I know was saved in the previous simulations.

@paulmueller It might well be that conversion is the same as in area deformation plots. But this is half-guessed.

@MartaUrb could you ask the guys for the data?
